.github/workflows/tests_preview.yml

```diff
@@ -17,6 +17,7 @@ on:
     paths:
       - "haystack/preview/**/*.py"
       - "test/preview/**/*.py"
+      - "e2e/preview/**/*.py"
```

Comment:
This doesn't seem right to me, we're not running end to end tests in this workflow. 🤔

Reply:
Oh. You're right!